clusterloader2/cmd/clusterloader.go

```diff
@@ -70,6 +67,19 @@ func validateFlags() *util.ErrorList {
 	return errList
 }
 
+func completeConfig(f *framework.Framework) error {
+	if clusterLoaderConfig.ClusterConfig.Nodes == 0 {
+		nodes, err := util.GetNodesNumber(f.GetClientSet())
+		if err == nil {
```

Comment:
The convention is:

if err != nil {
  return ...
}
// the rest

Reply:
Done.